source/ast/symbols/ClassSymbols.cpp

```diff
@@ -711,6 +711,13 @@ void ClassType::serializeTo(ASTSerializer& serializer) const {
         serializer.write("forward", *firstForward);
     if (genericClass)
         serializer.writeLink("genericClass", *genericClass);
+    auto syntax = getSyntax();
+    ASSERT(syntax);
+    auto& classSyntax = syntax->as<syntax::ClassDeclarationSyntax>();
```

Comment:
I don't think you want to be writing the stringified syntax here -- it means all consumers have to reimplement SystemVerilog parsing in order to understand the output. We've already done the work to get this into AST format, so you might as well write the rich structured information here instead. For example, call the getBaseClass() method and write that type (if non-null). Call getImplementedInterfaces() and serialize that as a JSON array, etc.